pyproject.toml

```diff
@@ -79,6 +79,14 @@ otel = [
     "opentelemetry-instrumentation-django",
     "opentelemetry-resource-detector-containerid",
     "opentelemetry-sdk",
+    "python-decouple>=3.8",
```

Comment:
I guess if we will use `config` in other modules, this should be part of the base dependencies?

Reply:
Dotenv files are only used in development, in prod they are injected as real environment variables.

Python dotenv loads the .env file and injects the parsed envvars in is environ. Python decouple doesn't, it treats the .env file as if it's is environ, but the OTEL SDK only reads from os.environ.

So any envvars you put in .env with only python-decouple will not be seen by tooling that only reads from is environ, which is *all* of our tooling.